docs/developing/autofocus.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Autofocus
+---
+
+<head>
+  <title>Autofocus</title>
+  <meta
+    name="description"
+    content="Learn how to manage focus in Ionic applications using the setFocus API instead of the autofocus attribute."
+  />
+</head>
+
+The `autofocus` attribute is a standard HTML attribute that allows developers to set focus to an element when a page loads. This attribute is commonly used to set focus to the first input element on a page. However, the `autofocus` attribute can cause issues in routing applications when navigating between pages. This is because the `autofocus` attribute will set focus to the element when the page loads, but will not set focus to the element when the page is revisited.
```

Comment:
This could probably be under its own heading since this documentation should be about managing focus in general, not just autofocus.